### PR TITLE
[BACKLOG-38705] - Adding kafka plugin enhancement changes in 10.0

### PR DESCRIPTION
**Pull request by lsaikrishna89**

### CODE REVIEW

**Comment by buildguy** [comment]
### :x: Build failed in 2m 36s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 25m 23s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **1278**, Failures: **0**, Skipped: **2** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/big-data-plugin/job/PR-2474/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 26m 45s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **1278**, Failures: **0**, Skipped: **2** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/big-data-plugin/job/PR-2474/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 24m 16s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **1278**, Failures: **0**, Skipped: **2** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/big-data-plugin/job/PR-2474/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: pom.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
     <dependency.high-scale-lib.revision>1.1.2</dependency.high-scale-lib.revision>
     <mondrian.version>10.0.0.0-SNAPSHOT</mondrian.version>
     <oss-licenses.version>10.0.0.0-SNAPSHOT</oss-licenses.version>
-    <pdi.version>10.0.0.0-SNAPSHOT</pdi.version>
+    <pdi.version>10.1.0.0-SNAPSHOT</pdi.version>
```
Comment (peterrinehart) [inline pom.xml:56]: I don't think you want to do this on the 10.0 branch

**Review by peterrinehart — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (lsaikrishna89) [reply pom.xml]: @peterrinehart thanks for reviewing the changes.
changed the version to 10.0 please, build is successful.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 26m 49s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **1278**, Failures: **0**, Skipped: **2** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/big-data-plugin/job/PR-2474/5/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 23m 33s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **1278**, Failures: **0**, Skipped: **2** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/big-data-plugin/job/PR-2474/6/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 23m 45s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **1278**, Failures: **0**, Skipped: **2** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/big-data-plugin/job/PR-2474/7/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 8m 54s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by lsaikrishna89** [comment]
creating other PR with only one commit.

**Pull request closed without merging**